packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/method_channel/method_channel_aggregate_query.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:cloud_firestore_platform_interface/src/method_channel/utils/source.dart';
+
+import 'method_channel_firestore.dart';
+import '../../cloud_firestore_platform_interface.dart';
+
+/// An implementation of [AggregateQueryPlatform] for the [MethodChannel]
+class MethodChannelAggregateQuery extends AggregateQueryPlatform {
+  MethodChannelAggregateQuery(QueryPlatform query) : super(query);
+
+  @override
+  Future<AggregateQuerySnapshotPlatform> get({
+    required AggregateSource source,
+  }) async {
+    final Map<String, dynamic>? data = await MethodChannelFirebaseFirestore
+        .channel
+        .invokeMapMethod<String, dynamic>(
+      'AggregateQuery#countGet',
```

Comment:
I'm not sure if we should have `AggregateQuery#countGet`.
I see the methods evolving to be able to chain 
```
query.count().avg().max().get()
```

Imo we should just have a get and send to native which aggregators we want to use, maybe by sending
```
{
   count: true
}
```

Reply:
I've updated to use `Aggregate#count`. We can update to whatever we want as and when it becomes more intricate on the SDKs. As it is underlying implementation, it will not affect the user facing API. So no need to worry about it for the time being.